charmcraft.yaml

```diff
@@ -29,7 +29,7 @@ parts:
       # Install PyYAML from binary and avoid building it from sources. This way, we can use PyYAML with C-optimized lib.
       # With the C-optimized lib, serialization in ops is 20x faster.
       - PyYAML
-      - cryptography
+      - cryptography==44.0.1
```

Comment:
This is a workaround to make the build work without re-syncing uv. It is obviated by https://github.com/canonical/grafana-agent-operator/pull/265 or can be separately fixed.

Reply:
Removed the version pinning after merging #265, will merge this one as soon as CI passes!